x-pack/plugins/canvas/public/application.tsx

```diff
@@ -159,17 +151,6 @@ export const initializeCanvas = async (
 export const teardownCanvas = (coreStart: CoreStart) => {
   destroyRegistries();
 
-  // Canvas pollutes the jQuery plot plugins collection with custom plugins that only work in Canvas.
-  // Remove them when Canvas is unmounted.
-  // see: ../canvas_plugin_src/renderers/plot/plugins/index.ts
-  if (includes($.plot.plugins, size)) {
-    remove($.plot.plugins, size);
-  }
-
-  if (includes($.plot.plugins, text)) {
-    remove($.plot.plugins, text);
-  }
-
```

Comment:
It seems like Canvas is still polluting the jQuery plot plugins in the [renderers/plot/index.ts file](https://github.com/elastic/kibana/blob/main/x-pack/plugins/canvas/canvas_plugin_src/renderers/plot/index.ts#L22-L28). These teardown functions were added in https://github.com/elastic/kibana/pull/90722 to fix [a problem rendering timelion charts after viewing Canvas](https://github.com/elastic/kibana/issues/90665). 

Even though timelion is deprecated, I'm slightly concerned about removing these teardowns. I don't have enough knowledge about flot and its usage in Canvas. So I might need to research this more. Maybe we should discuss offline when you have a chance @rshen91?

Reply:
Thank you for zooming - https://github.com/flot/flot/blob/8d8973ee34e80d277a878bfd256669418fc53b1c/src/jquery.flot.js